src/Xamarin.Android.Tools.ApiXmlAdjuster/Tests/TypeResolverTest.cs

```diff
@@ -18,19 +18,14 @@ public void SetupFixture ()
 		[Test]
 		public void TypeReferenceEquals ()
 		{
-			var intRef = new JavaTypeReference ("int");
-			Assert.AreEqual (intRef, new JavaTypeReference ("int"), "primitive types");
```

Comment:
Why are these tests invalidated? (*Presumably* they're invalidated, otherwise why would they be removed?)